help/include/sidebar_index.md

```diff
@@ -95,6 +95,7 @@
 * [Delete a message](/help/delete-a-message)
 * [Draft messages](/help/view-and-edit-your-message-drafts)
 * [Schedule a message](/help/schedule-a-message)
+* [Schedule a reminder](/help/schedule-a-reminder)
```

Comment:
Let's put this just below "Star a message"; I think it goes with that workflow-wise.